orchestrator/orchestrator/src/ethereum_event_watcher.rs

```diff
@@ -275,16 +276,16 @@ pub async fn get_block_delay<S: Signer>(eth_client: EthClient<S>) -> Result<U64,
     }
 
     match chain_id.unwrap() {
-        // Mainline Ethereum, Ethereum classic, or Ropsten, Mordor testnets
-        // all POW Chains
-        1 | 3 | 61 | 63 => Ok(13u8.into()),
+        // Mainline Ethereum, Ethereum classic, or the Ropsten, Kotti, Mordor testnets
+        // all Ethereum proof of stake Chains
+        1 | 3 | 6 | 7 => 96u8.into(),
         // Dev, our own Gravity Ethereum testnet, Hardhat
         // all non-pow chains
         2018 | 15 | 31337 => Ok(0u8.into()),
         // Rinkeby, Goerli, Kotti
         // Clique (POA) Consensus
-        4 | 5 | 6 => Ok(10u8.into()),
-        // assume the safe option (POW) where we don't know
-        _ => Ok(13u8.into()),
+        4 | 5 => Ok(10u8.into()),
+        // assume the safe option (POS) where we don't know
+        _ => 96u8.into(),
```

Comment:
there's a compilation error, because the expression expects a result: https://github.com/crypto-org-chain/gravity-bridge/actions/runs/3086869764/jobs/4991661491#step:5:1642

Reply:
thanks, fixed